### PR TITLE
add geowombat, arosics, geoarray

### PR DESCRIPTION
**Pull request by rluria14**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rluria14** [comment]
@ndmaxar @RyanHackney agree to be maintainers?

File: recipes/arosics/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - matplotlib
```
Comment (scopatz) [inline recipes/arosics/meta.yaml:23]: This  should be matplotlib-base

File: recipes/arosics/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
```
Comment (scopatz) [inline recipes/arosics/meta.yaml:22]: You have pip, so you should not have setuptools here

File: recipes/arosics/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    - matplotlib
+  run:
+    - cmocean
+    - folium >=0.6.0
+    - gdal
+    - geoarray >=0.8.30
+    - geojson
+    - geopandas
+    - matplotlib
```
Comment (scopatz) [inline recipes/arosics/meta.yaml:31]: This should be matplotlib-base

File: recipes/geoarray/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - dask >=0.15.0
+    - pip
+    - python
+    - matplotlib
```
Comment (scopatz) [inline recipes/geoarray/meta.yaml:23]: same

File: recipes/geoarray/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  run:
+    - dask >=0.15.0
+    - dill
+    - folium
+    - gdal >=2.1.0
+    - geojson
+    - matplotlib
```
Comment (scopatz) [inline recipes/geoarray/meta.yaml:30]: same

File: recipes/geowombat/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+    - llvm-openmp  # [osx]
+  host:
+    - pip
+    - python
+    - gdal
+    - cython
+  run:
+    - cython
+    - python
+    - matplotlib
```
Comment (scopatz) [inline recipes/geowombat/meta.yaml:29]: should be base

**Comment by rluria14** [comment]
@conda-forge/staged-recipes ready for final review

File: recipes/geowombat/yum_requirements.txt [hunk 7]
```diff
@@ -0,0 +1 @@
+mesa-libGL
```
Comment (chrisburr) [inline recipes/geowombat/yum_requirements.txt:1]: This probably isn't needed now matplotlib-base is being used

**Pull request merged**